src/types.ts

```diff
@@ -129,28 +129,34 @@ export interface FuseChannel {
 /** If `FuseChannel.rgbColor` is this value, it is disabled from fusion. */
 export const FUSE_DISABLED_RGB_COLOR = 0;
```

Comment:
Added comments here because I was getting confused by the accepted value range of each option